README.md

```diff
@@ -65,11 +84,23 @@ AZNFS is supported on following Linux distros:
 
 ## Usage Instructions
 
-- Mount the Azure Blob NFS share using following command:
+- Mount the Azure Blob NFSv3 share using following command:
 	```
 	sudo mount -t aznfs -o vers=3 <account-name>.blob.core.windows.net:/<account-name>/<container-name> /mountpoint
 	```
-- Logs generated from AZNFS can be found in `/opt/microsoft/aznfs/data/aznfs.log`.
+- Mount the Azure File NFSv4 share using following command:
+	```
+	sudo mount -t aznfs -o vers=4.1 <account-name>.file.core.windows.net:/<account-name>/<container-name> /mountpoint
+	```
+   For isolated environments, ensure that the environment variable "AzureEndpointOverride" is set to the appropriate endpoint before running the mount command:
+	```
+	export AzureEndpointOverride=".example.end.point"
+	```
```

Comment:
Is there some real customer scenario we are targeting using this env variable support, or is it just being flexible.
Also, is that scenario only possible in our testing or even possible at customers?

Reply:
This if for the air-gapped tenants.